src/test/java/org/sagebionetworks/bridge/validators/ActivityEventValidatorTest.java

```diff
@@ -47,7 +47,6 @@ public void eventIdNull() {
         assertValidatorMessage(INSTANCE, event, "eventId", EVENT_ID_ERROR);
     }
 
-    @Test
```

Comment:
Was this test intentionally removed?

Reply:
No. Added it back. :-( I may have been contemplating removing this event type since it's never been used, but it's still part of our API.